src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerCFTTest.java

```diff
@@ -260,7 +260,7 @@ public void should_return_a_200_and_retrieve_a_task_by_event_and_case_match() {
             .body("tasks[0].type", equalTo("reviewTheAppeal"))
             .body("tasks[0].jurisdiction", equalTo("IA"))
             .body("tasks[0].case_type_id", equalTo("Asylum"))
-            .body("tasks[0].permissions.values", hasItems("Read", "Refer", "Own", "Manage", "Cancel"))
+            .body("tasks[0].permissions.values", hasItems("Read", "Refer", "Own"))
```

Comment:
as per requirement, we will need to change the matcher to check this contains only the permissions relevant to the user

Reply:
Added as part of new commit